Mathlib/MeasureTheory/Measure/Stieltjes.lean

```diff
@@ -492,3 +492,46 @@ lemma eq_of_measure_of_eq (g : StieltjesFunction) {y : ℝ}
       exact f.mono hxy
 
 end StieltjesFunction
+
+variable (μ : Measure ℝ) [IsFiniteMeasureOnCompacts μ]
+
+lemma tendsto_measure_Icc_nhdsWithin_gt (b : ℝ) :
+    Tendsto (fun δ ↦ μ (Icc (b - δ) (b + δ))) (𝓝[>] (0 : ℝ)) (𝓝 (μ {b})) := by
+  have : {b} = ⋂ (r > 0), Icc (b - r) (b + r) := by
+    apply Subset.antisymm
+    · simp only [gt_iff_lt, subset_iInter_iff, singleton_subset_iff, mem_Icc, tsub_le_iff_right,
+        le_add_iff_nonneg_right, and_self]
+      exact fun i hi ↦ hi.le
+    · intro x hx
+      simp only [gt_iff_lt, mem_iInter, mem_Icc, tsub_le_iff_right] at hx
+      simp only [mem_singleton_iff]
+      apply le_antisymm
+      · exact _root_.le_of_forall_pos_le_add (fun r hr ↦ (hx r hr).2)
+      · exact _root_.le_of_forall_pos_le_add (fun r hr ↦ (hx r hr).1)
+  rw [this]
+  apply tendsto_measure_biInter_gt (fun r hr ↦ measurableSet_Icc)
+  · intro r s _rpos hrs
+    apply Icc_subset_Icc (by linarith) (by linarith)
+  · refine ⟨1, zero_lt_one, ?_⟩
+    exact isCompact_Icc.measure_ne_top
+
+lemma tendsto_measure_Icc_nhdsWithin_ge (b : ℝ) :
+    Tendsto (fun δ ↦ μ (Icc (b - δ) (b + δ))) (𝓝[≥] (0 : ℝ)) (𝓝 (μ {b})) := by
+  have : 𝓝[≥] (0 : ℝ) = 𝓝[>] (0 : ℝ) ⊔ 𝓝[{0}] 0 := by
+    simp only [union_singleton, Ioi_insert, ← nhdsWithin_union]
```

Comment:
This should definitely be a lemma in its own right - I'm surprised it isn't already (both the literal statement of the `have`, and the consequence of interest, which is that `Tendsto f (𝓝[≥] a) (𝓝 b)` iff `Tendsto f (𝓝[>] a) (𝓝 b)` and `f a = b`).